replicate/helpers.py

```diff
@@ -43,7 +43,7 @@ def encode_json(
             return encode_json(file, client, file_encoding_strategy)
     if isinstance(obj, io.IOBase):
         if file_encoding_strategy == "base64":
-            return base64.b64encode(obj.read()).decode("utf-8")
```

Comment:
lolwut. how did this ever work? did it just... not?

Reply:
It did not.